numpy/linalg/umath_linalg.c.src

```diff
@@ -861,6 +865,7 @@ nan_@TYPE@_matrix(void *dst_in, const LINEARIZE_DATA_t* data)
     }
 }
 
+#if 0
 static NPY_INLINE void
 zero_@TYPE@_matrix(void *dst_in, const LINEARIZE_DATA_t* data)
```

Comment:
Could we put `__attribute__((unused))` on these instead? `#if 0` code is in danger of compilation bitrot.

Reply:
does msvc have an __attribute__((unused)) equivalent?